deployer/deployer-core/src/main/java/io/hyscale/deployer/core/model/AppMetadata.java

```diff
@@ -22,6 +22,7 @@
 
     private String namespace;
     private String appName;
+    private String envName;
```

Comment:
An App can have multiple environments right?

Reply:
But not more than a single app can be deployed in a namespace